Library/Homebrew/dev-cmd/pr-pull.rb

```diff
@@ -291,34 +302,49 @@ def cherry_pick_pr!(user, repo, pr, args:, path: ".")
     Utils::Git.cherry_pick!(path, "--ff", "--allow-empty", *commits, verbose: args.verbose?, resolve: args.resolve?)
   end
 
-  def formulae_need_bottles?(tap, original_commit, user, repo, pr, args:)
+  def packages_need_bottles?(tap, original_commit, user, repo, pr, args:)
     return if args.dry_run?
 
     labels = GitHub.pull_request_labels(user, repo, pr)
 
     return false if labels.include?("CI-syntax-only") || labels.include?("CI-no-bottles")
 
-    changed_formulae(tap, original_commit).any? do |f|
-      !f.bottle_unneeded? && !f.bottle_disabled?
+    changed_packages(tap, original_commit).any? do |f|
+      !f.instance_of?(Cask::Cask) && !f.bottle_unneeded? && !f.bottle_disabled?
```

Comment:
Don't think this change is needed given request above.

Reply:
Still think this function should be named `formulae_need_bottles?` because casks never do and consider adding a `changed_formulae` function that strips out all casks.

Reply:
Yeah, I'll keep the naming. But the check should stay here IMO